LayerClient/LayerClient.py

```diff
@@ -319,6 +320,25 @@ def send_announcement(self, sender, recipients, message_parts,
             )
         )
 
+    def mark_delivery_receipt_message(self, message_uuid='', type='sent'):
+        """
+        Mark delivery receipt for message
+
+        Parameters:
+        - `message_uuid`: The uuid of the message read
+		- `type`: receipt message type
+        """
+        request_data = {
+            'type': type
+        }
+        self._raw_request(
+            METHOD_POST,
+            self._get_layer_uri(
+                LAYER_URI_RECEIPTS,
```

Comment:
This will generate the URI `/receipts/MESSAGE_ID`, which I don't think is correct - I think this should instead be

    self._get_layer_uri(
        LAYER_URI_MESSAGES,
        message_uuid,
        LAYER_URI_RECEIPTS
    )

which should give you `/messages/MESSAGE_ID/receipts` as per the docs